Dockerfile

```diff
@@ -1,11 +1,11 @@
-# NOTE: remove the "-slim" if you need a full featured base image.
-FROM python:3.11-slim
+FROM python:3.9
```

Comment:
Is there a reason you went back to python 3.9?

Reply:
Yes.  I can't recall now what it was but there is something in 3.9 that was removed in 3.11.